app/components/progressive_image/progressive_image.test.js

```diff
@@ -2,7 +2,7 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import {shallow} from 'enzyme';
+import {shallowWithIntl} from 'test/intl-test-helper';
```

Comment:
why this change? as far as I can tell the ProgressiveImage component does not use intl